http_server/queries/bans/ban_lift.php

```diff
@@ -16,7 +16,7 @@ function ban_lift($pdo, $ban_id, $lifted_by, $lifted_reason)
 
     $result = $stmt->execute();
     if ($result === false) {
-        throw new Exception('Could not lift ban');
+        throw new Exception("Could not lift ban #$ban_id.");
```

Comment:
If this is echoing out to an html page, it possibly needs htmlspecialchars?

Reply:
I can add an integer type conversion.